hoover/search/es.py

```diff
@@ -43,13 +64,59 @@ def get(collection_id, doc_id):
         id=doc_id,
     )
 
-
-def search(query, fields, highlight, collections, from_, size, sort, aggs):
+def _get_indices(collections):
     from .models import Collection
     indices = ','.join(
         c.index for c in
         Collection.objects.filter(name__in=collections)
     )
+    return indices
+
+@convert_es_errors
+def batch_count(query_strings, collections, aggs=None):
+    def _serialize(object):
+        if object:
+            return json.dumps(
+                object,
+                separators=(',', ':'),
+            ).replace('\n', '')
+        return '{}'
+
+    def _build_query_lines(meta, query_string, aggs):
+        query = {
+            "query": {
+                "query_string": {
+                    "query": query_string,
```

Comment:
Yeah, but will those filters be different for each query involved? I think we should extend this when needed to include a `filter` optional param that will be the same for each query.

Moreover, passing an `query_string` will make sending it back simple, so the UI will only use the server reply for outputting the results (instead of storing them in the internal UI state and matching them up again after the fact.

In any case, I've implemented this as you requested.